inc/nu/impl/proclet.ipp

```diff
@@ -73,6 +79,26 @@ retry:
     Migrator::migrate_thread_and_ret_val<void>(
         std::move(return_buf), to_proclet_id(caller_header), nullptr, nullptr);
   }
+  else{
+    // metric logging
+    if (caller_header){
```

Comment:
I think this "if" is not needed

Reply:
In testing I encountered cases where caller_header is null, which causes errors without the if statement. It may be possible that it only happens in invoke_remote_with_ret though.